firefox-ios/Client/Frontend/Home/Bookmarks/BookmarksViewModel.swift

```diff
@@ -111,7 +111,7 @@ extension BookmarksViewModel: HomepageViewModelProtocol, FeatureFlaggable {
     }
 
     var isEnabled: Bool {
-        return featureFlags.isFeatureEnabled(.hntBookmarksSection, checking: .userOnly)
+        return !featureFlags.isFeatureEnabled(.homepageStoriesRedesign, checking: .buildOnly)
```

Comment:
nit: maybe we can improve the comment in the yaml file that this will also include content removal?
`If true, enables the stories section redesign on homepage, which also includes content removal of other sections (i.e. jump back in, bookmarks)`